queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE %mon;
+SELECT * FROM animals WHERE date_of_birth BETWEEN CAST('2016-01-01' AS DATE) AND CAST('2019-12-30' AS DATE);
```

Comment:
- [ ] Kindly make sure that the end date is 31 Dec. Currently is 30 Dec.
- [ ] Make sure to add the updated screenshot for this query.

Reply:
- [ ] As the previous reviewer mentioned, Kindly change the end date to `2019-12-31` instead of `2019-12-30` 💯